### ISSUE TITLE
Convert to react

### ISSUE THREAD

**Issue by aditeyaS** [aditeyaS/peoples-choice#95]
Convert it to a simple react app to make it easier to implement TMDB's API. You can omit all the fancy stuff like animation and all.

**Comment by imsherlocked** [comment]
Hey @aditeyaS can I work on this issue?

**Comment by aditeyaS** [comment]
Sure @imsherlocked . I will assign it to you

**Comment by imsherlocked** [comment]
@aditeyaS  do you need me to provide a version using react for the current project?

**Comment by aditeyaS** [comment]
Yes, that's what this issue was about. If we convert it to react it will be easier to integrate TMDB API
